SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -786,7 +800,7 @@ void USpatialNetDriver::BeginDestroy()
 			{
 				Cast<USpatialGameInstance>(LocalWorld->GetGameInstance())->DestroySpatialWorkerConnection();
 			}
-
+			Connection->UnbindCallbacks();
```

Comment:
This is a bit weird. Can it not just unbind it's own callbacks.

Reply:
Or can this not be part of the instance destroying the connection above?

Reply:
Removed.